.appveyor.yml

```diff
@@ -0,0 +1,18 @@
+environment:
+  matrix:
+    - RUBY_VERSION: "25-x64"
```

Comment:
@Watson1978 do you think you can get it to pass on Ruby 2.3 and IM 6.8?

Reply:
I want to test that it works with https://github.com/rmagick/rmagick/pull/315 and the PR effect to Ruby 2.4+.

Ruby 2.4 and IM 6.8 is better

Reply:
Hmm, IM 6.8 is too old and it cause a failure to build :(
https://ci.appveyor.com/project/mockdeep/rmagick/builds/22085425

Reply:
Hmm, why would `GetExceptionInfo` be missing on Windows? These compile fine on Linux. Looking at the configured options, one thing that stands out to me is that it is missing `:magick_config` and `:local_libs`. Does that seem significant to you?

Appveyor:

```rb
Configured compile options: {
  :magick_config=>nil,
  :with_magick_wand=>nil,
  :pkg_config=>nil,
  :magick_version=>"6.8.9",
  :local_libs=>"",
  :cflags=>"-march=x86-64 -mtune=generic -O2 -pipe",
  :cppflags=>"-I\"C:\\Program Files\\ImageMagick-6.8.9-Q16/include\"",
  :ldflags=>"-L\"C:\\Program Files\\ImageMagick-6.8.9-Q16/lib\"",
  :defs=>[],
  :config_h=>"Makefile rmagick.h"
}
```

TravisCI

```rb
Configured compile options: {
  :magick_config=>true,
  :with_magick_wand=>false,
  :pkg_config=>false,
  :magick_version=>"6.8.9",
  :local_libs=>" -lMagickCore-6.Q16  ",
  :cflags=>" -fopenmp -DMAGICKCORE_HDRI_ENABLE=0 -DMAGICKCORE_QUANTUM_DEPTH=16 -I/usr/include/ImageMagick-6  ",
  :cppflags=>" -fopenmp -DMAGICKCORE_HDRI_ENABLE=0 -DMAGICKCORE_QUANTUM_DEPTH=16 -I/usr/include/ImageMagick-6  ",
  :ldflags=>" -lMagickCore-6.Q16  ",
  :defs=>[],
  :config_h=>"Makefile rmagick.h"
}
```

Reply:
By some reason, `GetExceptionInfo()` API was not contained in library of old ImageMagick.